.circleci/config.yml

```diff
@@ -52,6 +52,7 @@ jobs:
         root: .
         paths:
         - ./architect
+        - ./cert-exporter
```

Comment:
This will persist the (already templated) chart to the workspace, so we can use it in the publishing call.